kivy/uix/label.py

```diff
@@ -303,8 +303,9 @@ class Label(Widget):
 
     _font_properties = ('text', 'font_size', 'font_name', 'bold', 'italic',
                         'underline', 'strikethrough', 'font_family', 'color',
-                        'disabled_color', 'halign', 'valign', 'padding_x',
-                        'padding_y', 'outline_width', 'disabled_outline_color',
+                        'disabled_color', 'halign', 'valign',
+                        'padding', 'padding_x', 'padding_y',
```

Comment:
At line **345**:
```python
dkw = {x: getattr(self, x) for x in self._font_properties}
```

`padding_x` and `padding_y` are accessed even if the user did not specify them.
This is (correctly) printing a deprecation warning.

We should avoid to pass `padding_x` and `padding_y` to `CoreLabel` or `CoreMarkupLabel`, since are deprecated also of on `CoreLabel` and `CoreMarkupLabel` side.

The suggestion here is to remove `padding_x` and `padding_y` from `_font_properties`, and "manually" bind for changes (we are doing the same for `disabled` property)